src/gui/optionsdialog.ui

```diff
@@ -4039,5 +4062,21 @@ Use ';' to split multiple entries. Can use wildcard '*'.</string>
     </hint>
    </hints>
   </connection>
+  <connection>
```

Comment:
For new code it is preferred to be connected in .cpp file.

Reply:
Gotcha. Moved it to the cpp